Sources/Tracking/Store/SdkConfig.swift

```diff
@@ -18,6 +18,13 @@ public struct SdkConfig {
      */
     public var autoTrackPushEvents: Bool = true
 
+    /**
+     Number of tasks in the background queue before the queue begins operating.
+     This is mostly used during development to test configuration is setup. We do not recommend
+     modifying this value because it impacts battery life of mobile device.
+     */
+    public var backgroundQueueMinNumberOfTasks = 10
```

Comment:
I noticed that's what Segment uses in it's SDK so I felt it was a good default. [As mentioned ](https://github.com/customerio/customerio-ios/pull/78#discussion_r753333144), there are more ways the SDK can run the queue to prevent tasks getting stale. 

The queue is great because it allows us to retry HTTP requests for customers but it's also great because it will improve battery life. The more that we bulk running queue tasks, the better the battery will be which is why this value is greater then lets say 3 or 5.